lib/ui/scripts/script.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import webpack from 'webpack';
+
+import config from './config';
+
+const resolveLocal = dir => path.join(__dirname, dir);
+const webpackAsPromised = c =>
+  new Promise((res, rej) => {
+    webpack(c).run((err, stats) => {
+      if (err || stats.hasErrors() || stats.hasWarnings()) {
+        rej(stats);
+      }
+      res(stats);
+    });
+  });
+
+const run = async () => {
+  const out2 = await webpackAsPromised(
```

Comment:
this function can just return the promise instead of awaiting for it. the only reason you'd await in this function is if you wanted specific errors in the try/catch